src/components/v2v3/shared/FundingCycleConfigurationDrawers/NftDrawer/index.tsx

```diff
@@ -53,22 +61,51 @@ export default function NftDrawer({
     theme,
     theme: { colors },
   } = useContext(ThemeContext)
+
+  const {
+    nftRewards: { rewardTiers: initialRewardTiers },
+  } = useContext(NftRewardsContext)
+
   const dispatch = useAppDispatch()
   const {
-    nftRewards: { rewardTiers: savedRewardTiers },
+    nftRewards,
     projectMetadata: { name: projectName, logoUri, infoUri },
   } = useAppSelector(state => state.editingV2Project)
 
+  const reduxRewardTiers = useMemo(() => nftRewards.rewardTiers, [nftRewards])
+
+  const { signer } = useWallet()
+
+  const { fundingCycleMetadata } = useContext(V2V3ProjectContext)
+
   const [marketplaceForm] = useForm<MarketplaceFormFields>()
   const [postPayModalForm] = useForm<NftPostPayModalFormFields>()
 
   const [addTierModalVisible, setAddTierModalVisible] = useState<boolean>(false)
   const [submitLoading, setSubmitLoading] = useState<boolean>(false)
 
   const [rewardTiers, setRewardTiers] = useState<NftRewardTier[]>(
-    savedRewardTiers ?? [],
+    reduxRewardTiers ?? [],
   )
 
+  useEffect(() => {
+    if (!rewardTiers.length) {
+      setRewardTiers(reduxRewardTiers ?? [])
+    }
+  }, [reduxRewardTiers, rewardTiers])
+
+  // a list of the `tierRanks` (IDs) of tiers that have been edited
+  const [editedRewardTierIds, setEditedRewardTierIds] = useState<number[]>([])
```

Comment:
**suggestion (non blocking)** its not caught by eslint but we should try order components in the following way:

```
<context>
<state>
<hooks>
```

Reply:
Got it.